<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main Window Style */
QMainWindow {
    background-color: #1e1e1e; /* Dark background */
    color: #ffffff; /* White text for all elements */
    border: 1px solid #444;
}

/* Buttons */
QPushButton {
    background-color: #005f73; /* Arduino blue */
    color: #ffffff; /* White text */
    border: 2px solid #003f5c;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #0a9396; /* Lighter blue for hover effect */
}

QPushButton:pressed {
    background-color: #003f5c; /* Darker blue when pressed */
    border: 2px solid #005f73;
}

/* Labels */
QLabel {
    color: #ffffff; /* White text */
    font-size: 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffff;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="motorStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="accessGrantedLED">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCode">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>401</width>
      <height>321</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>currentTime</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>user</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status Of Acces</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
